.github/actions/get-jtreg/action.yml

```diff
@@ -56,8 +56,14 @@ runs:
 
     - name: 'Build JTReg'
       run: |
+        # If runner architecture is x64 set JAVA_HOME_17_X64 otherwise set to JAVA_HOME_17_arm64
+        if [[ '${{ runner.arch }}' == 'X64' ]]; then
+          JDK="$JAVA_HOME_17_X64"
+        else
+          JDK="$JAVA_HOME_17_arm64"
+        fi
         # Build JTReg and move files to the proper locations
-        bash make/build.sh --jdk "$JAVA_HOME_17_X64"
+        bash make/build.sh --jdk "$JDK"
```

Comment:
We did this `JAVA_HOME_17_X64` change for https://github.com/openjdk/jdk/commit/959a61fdd483c9523764b9ba0972f59ca06db0ee, which is partially reverted during upgrade of GH actions. (...and I forgot to revert this hunk, apparently.) I wonder if it would be simpler if we switch back to boot JDK for building jtreg again, which would eliminate this hunk? Might make backports easier too. 

I have no strong opinion here, though.

Reply:
yeah this does make sense, anything to remove a horrible if statement 👍🏼

Reply:
@shipilev updated PTAL